set.mm

```diff
@@ -759865,61 +759865,58 @@ this concept is not fully developed yet (it is within a mathbox), and it is
        14-May-2024.) $)
-    naryfvalel $p |- ( ( X e. V /\ N e. NN0 )
-                       -> ( F e. ( X NaryF N ) <-> F : ( X ^m I ) --> X ) ) $=
-      ( wcel cn0 wa cnaryf co cmap wf wceq naryfval adantl eleq2d cvv wb ovexd
-      elmapg sylan2 bitrd ) EDGZCHGZIZAECJKZGAEEBLKZLKZGZUHEAMZUFUGUIAUEUGUINUD
-      BCEFOPQUEUDUHRGUJUKSUEEBLTEUHADRUAUBUC $.
+    naryfvalel $p |- ( ( N e. NN0 /\ X e. V )
+                       -> ( F e. ( N -aryF X ) <-> F : ( X ^m I ) --> X ) ) $=
+      ( cn0 wcel cnaryf co cmap wf naryfval eleq2d cvv wb elmapg mpan2 sylan9bb
+      ovex ) CGHZACEIJZHAEEBKJZKJZHZEDHZUCEALZUAUBUDABCEFMNUFUCOHUEUGPEBKTEUCAD
+      OQRS $.
 
     $( Reverse closure for n-ary (endo)functions.  (Contributed by AV,
        14-May-2024.) $)
-    naryrcl $p |- ( F e. ( X NaryF N ) -> ( X e. _V /\ N e. NN0 ) ) $=
-      ( vx vn cvv cn0 cv cc0 cfzo co cmap cnaryf df-naryf elmpocl ) FGHIFJZRKGJ
-      LMNMNMDCOAFGPQ $.
+    naryrcl $p |- ( F e. ( N -aryF X ) -> ( N e. NN0 /\ X e. _V ) ) $=
+      ( vx vn cn0 cvv cv cc0 cfzo co cmap cnaryf df-naryf elmpocl ) FGHIGJZRKFJ
+      LMNMNMCDOAGFPQ $.
 
     $( The value of an n-ary (endo)function on a set ` X ` is an element of
        ` X ` .  (Contributed by AV, 14-May-2024.) $)
-    naryfvalelfv $p |- ( ( F e. ( X NaryF N ) /\ A : I --> X )
+    naryfvalelfv $p |- ( ( F e. ( N -aryF X ) /\ A : I --> X )
                          -> ( F ` A ) e. X ) $=
-      ( cvv wcel cn0 wa cnaryf co wf cfv naryrcl adantr cmap wi naryfvalel w3a
-      simp2 simpl cc0 cfzo a1i elmapd biimpar 3adant2 ffvelrnd 3exp sylbid impd
-      ovexi mpcom ) EGHZDIHZJZBEDKLHZCEAMZJABNEHZURUQUSBCDEFOPUQURUSUTUQURECQLZ
-      EBMZUSUTRBCDGEFSUQVBUSUTUQVBUSTVAEABUQVBUSUAUQUSAVAHZVBUQVCUSUQECAGGUOUPU
-      BCGHUQCUCDUDFUMUEUFUGUHUIUJUKULUN $.
+      ( cnaryf co wcel wf wa cmap cn0 cvv naryrcl naryfvalel biimpd mpcom simpr
+      adantr cc0 cfzo ovexi a1i elmapd biimpar sylan ffvelrnd ) BDEGHIZCEAJZKEC
+      LHZEABUIUKEBJZUJDMIZENIZKZUIULBCDEFOZUOUIULBCDNEFPQRTUIUOUJAUKIZUPUOUQUJU
+      OECANNUMUNSCNIUOCUADUBFUCUDUEUFUGUH $.
   $}
 
   ${
     $d N w $.  $d V w $.  $d X w $.
     $( An n-ary (endo)function on a set ` X ` expressed as a function over the
        set of words on ` X ` of length ` n ` .  (Contributed by AV,
        4-Jun-2024.) $)
-    naryfvalelwrdf $p |- ( ( X e. V /\ N e. NN0 ) -> ( F e. ( X NaryF N )
+    naryfvalelwrdf $p |- ( ( N e. NN0 /\ X e. V ) -> ( F e. ( N -aryF X )
                            <-> F : { w e. Word X | ( # ` w ) = N } --> X ) ) $=
-      ( wcel cn0 wa cnaryf co cc0 cfzo cmap wf cv chash cfv wceq cword crab
-      eqid naryfvalel wrdnval eqcomd feq2d bitrd ) EDFCGFHZBECIJFEKCLJZMJZEBNAO
-      PQCRAESTZEBNBUHCDEUHUAUBUGUIUJEBUGUJUIACEDUCUDUEUF $.
+      ( cn0 wcel wa cnaryf co cc0 cfzo cmap wf cv chash cfv wceq cword crab
+      eqid naryfvalel wrdnval ancoms feq2d bitr4d ) CFGZEDGZHZBCEIJGEKCLJZMJZEB
+      NAOPQCRAESTZEBNBUJCDEUJUAUBUIULUKEBUHUGULUKRACEDUCUDUEUF $.
```

Comment:
it looks like it gets longer for no reason, but really there are two changes:
* ancoms (needed since the antecedent was commuted, or if the antecedent wasn't commuted, it would be since naryfvalel doesn't match wrdnval)
* bytes saved from using bitr4d (and avoiding eqcomd)